pkg/kubelet/config/http.go

```diff
@@ -129,7 +129,7 @@ func (s *sourceURL) extractFromURL() error {
 			// It parsed but could not be used.
 			return multiPodErr
 		}
-		pods := make([]*v1.Pod, 0)
+		pods := make([]*v1.Pod, 0, len(podList.Items))
```

Comment:
The second parameter of make indicates the initial capacity. I think we should change it to this:
 `pods := make([]*v1.Pod, len(podList.Items))`

Reply:
make(type, len, cap)

Reply:
https://github.com/golang/go/blob/2ebe77a2fda1ee9ff6fd9a3e08933ad1ebaea039/src/builtin/builtin.go#L172-L189

Reply:
Sorry, I misunderstood. You're right.